cql3/query_processor.cc

```diff
@@ -1179,4 +1179,39 @@ void query_processor::reset_cache() {
     _authorized_prepared_cache.reset();
 }
 
+future<> query_processor::truncate_table(sstring ks_name, sstring table_name, std::chrono::milliseconds timeout_in_ms) {
+    auto ks = db().find_keyspace(ks_name);
+    if (ks.get_replication_strategy().uses_tablets()) {
+        log.debug("Running TRUNC TABLE (with tablets) on {}.{}", ks_name, table_name);
```

Comment:
nit: please do not abbreviate TRUNCATE to TRUNC

Reply:
I removed the abbreviations.